meilisearch/src/analytics/segment_analytics.rs

```diff
@@ -720,6 +729,11 @@ impl SearchAggregator {
             let used_syntax = self.used_syntax.entry(key).or_insert(0);
             *used_syntax = used_syntax.saturating_add(value);
         }
+
+        // attributes_to_search_on
+        self.attributes_to_search_on_total_number_of_use +=
+            other.attributes_to_search_on_total_number_of_use;
```

Comment:
other additions in this file are using `saturating_add`

```suggestion
        self.attributes_to_search_on_total_number_of_use = self
            .attributes_to_search_on_total_number_of_use
            .saturating_add(other.attributes_to_search_on_total_number_of_use);
```